src/v2/view-builder/views/shared/polling.js

```diff
@@ -2,8 +2,9 @@ import { _ } from 'okta';
 
 export default {
   startPolling () {
+    this.pollingInterval = this.options.currentViewState.refresh;
```

Comment:
Why doesn't this have the calculation like
`this.pollingInterval = Math.ceil(this.options.appState.getCurrentViewState().refresh / 1000);`
from
 https://github.com/okta/okta-signin-widget/pull/1563/files#diff-09e81d05f4347e1aeffa0830d7f8f711b4496f57f24cc51ed7331b0932aef815R38

Reply:
Good question.. because the one in PollingView shows up as seconds to enduser.. We don't want to show that in `ms`

Reply:
Ah I get it.